spec/controllers/verify/finance_controller_spec.rb

```diff
@@ -141,6 +141,8 @@
 
     context 'when the form is invalid' do
       it 'tracks the form errors and does not make a vendor API call' do
+        expect(Idv::FinancialsValidator).to_not receive(:new)
```

Comment:
I can do that. We've been using both styles. Are we saying we should pick one going forward and always group expectations?